client/extensions/woocommerce/state/data-layer/request/actions.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { uniqueId } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	WOOCOMMERCE_REQUEST,
+} from 'woocommerce/state/action-types';
+
+function _createRequestAction( method, siteId, path, body ) {
+	return {
+		type: WOOCOMMERCE_REQUEST,
+		requestId: uniqueId( 'request_' ),
+		method,
+		siteId,
+		path,
+		body,
+	};
+}
+
+export function get( siteId, path ) {
+	return _createRequestAction( 'get', siteId, path );
+}
+
+export function post( siteId, path, body ) {
+	return _createRequestAction( 'post', siteId, path, body || {} );
```

Comment:
The `body || {}` logic is already present in the `request.js` file. No need to do it here, unless you feel it adds some semantic clarity.